### ISSUE TITLE
changing login credentials

### ISSUE THREAD

**Issue by rsese** [atom/github#1621]
_From @iraizo on July 26, 2018 14:13_

Today i changed my github name and now i cant push updates since the login credentials are outdated any way to update them?

_Copied from original issue: atom/atom.io#161_

**Comment by rsese** [comment]
Thanks for reaching out - just to clarify, when you say "can't push updates", do you mean that you're having trouble pushing commits using the [GitHub package](https://github.com/atom/github/)?

**Comment by rsese** [comment]
_From @iraizo on July 30, 2018 10:26_

yes

**Comment by rsese** [comment]
Thanks for confirming @iraizo - it sounds like your credentials are saved so you would need to update or delete your saved credentials.  What operating system are you on?  If you're on macOS for example, you can follow this help article:

https://help.github.com/articles/updating-credentials-from-the-osx-keychain/

**Comment by larrykollar** [comment]
How to do this on WIndows? I access an internal BitBucket repository, and we have to update our passwords every few months. I'm able to access the repository using Firefox, but in the past Atom would (at some point) pop up a dialog allowing me to re-enter my new password. It hasn't happened this time after multiple attempts, quitting Atom and restarting it, trying Fetch instead of Push. How can I tell Atom to display the dialog? I went through all the config screens, and the keybinding screens, and didn't find anything.

**Comment by iraizo** [comment]
@larrykollar https://discuss.atom.io/t/atom-want-to-push-with-old-github-credentials/57486 maybe this will help you?

**Comment by larrykollar** [comment]
No, but another tab I'd opened while trying to run down the problem pointed me in the right direction. Windows 10 has Credential Manager, which allowed me to update the password for the web site. Once I did that, the push slid right on through.

Maybe this will help someone else.

**Comment by ClementGre** [comment]
I find a solution : I unlog from GitHub in Atom, then i add the remote in command line and I can push After that, I re-login me with a token, on Atom, and it's work ! 
![image](https://user-images.githubusercontent.com/51322714/81598014-e996c000-93c6-11ea-9e5e-e9f5594f6e39.png)

**Comment by howardrork** [comment]
On windows this helped me.

https://stackoverflow.com/questions/15381198/remove-credentials-from-git
